worker/apiserver/worker.go

```diff
@@ -112,12 +112,16 @@ func NewWorker(config Config) (worker.Worker, error) {
 		return nil, errors.Annotate(err, "cannot fetch the controller config")
 	}
 
+	metricsCollector := apiserver.NewMetricsCollector(apiserver.SubsystemNamespace)
+	deprecatedMetricsCollector := apiserver.NewMetricsCollector(apiserver.DeprecatedSubsystemNamespace)
```

Comment:
We don't want copies of all metrics, just one of them.

Reply:
No problemo. Will sort it out.